src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -6,8 +6,14 @@
     <span class="quick-add-label">Create Work Item</span>
   </div>
   <div class="flex-container flex-grow-1 addWorkItemBlk " *ngIf="showQuickAdd">
-    <div class="workItemQuickAdd_IconType btn btn-default">
-      <i class="fa fa-bookmark pull-left "></i>
+    <!-- type dropdown -->
+    <div class="dropup dropdown-kebab-pf" dropdown>
+      <button class="btn btn-link dropdown-toggle" data-toggle="dropdown" type="button" dropdownToggle>
+        <i class="fa pull-left" [ngClass]="selectedType?selectedType.attributes.icon:''"></i>
+      </button>
+      <ul class="dropdown-menu" aria-labelledby="dropdownKebab" role="menu" dropdownMenu>
```

Comment:
add a `margin-left: 8px;` to push the menu over so that the caret is right above the icon